owl/EASE-OBJ.owl

```diff
@@ -240,6 +240,18 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#hasFeature -->
+
+    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#hasFeature">
+        <rdfs:subPropertyOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#hasConstituent"/>
```

Comment:
I would lean towards hasPart as a superproperty, but I guess the idea with hasPart is that it points to entities that can be taken out of the whole and have independent existence, while these features do not?